zgw/object-management/src/test/kotlin/com/ritense/objectmanagement/service/ObjectManagementServiceIntTest.kt

```diff
@@ -61,20 +107,167 @@ internal class ObjectManagementServiceIntTest: BaseIntegrationTest() {
 
     @Test
     @Order(4)
-    fun deleteById() {
-       val objectManagement = createObjectManagement()
+    fun `delete by id`() {
+        val objectManagement = createObjectManagement()
 
         objectManagementService.deleteById(objectManagement.id)
 
         val objectManagementList = objectManagementService.getAll()
         assertThat(!objectManagementList.contains(objectManagement))
     }
 
+    @Test
+    @Order(5)
+    fun `get objects from objects api with search fields`() {
+        val objectUrl = mockApi.url("/some-object").toString()
+        val objectTypesApiUrl = mockApi.url("/some-objectTypesApi").toString()
+
+        val authenticationPlugin = pluginService.createPluginConfiguration(
+            title = "Objecten authentication",
+            properties = ObjectMapper().readTree(
+                "{\"token\":\"some-secret-token\"}," +
```

Comment:
I believe this is an invalid json. And please use [raw strings](https://kotlinlang.org/docs/strings.html#raw-strings) for readability